includes/Core/Key_Metrics/Key_Metrics_Settings.php

```diff
@@ -49,6 +49,37 @@ protected function get_default() {
 		return array();
 	}
 
+	/**
+	 * Merges an array of settings to update. Only existing keys will be updated.
```

Comment:
This JSDoc should also be updated:
```suggestion
	 * Merges an array of settings to update.
```